client/dist/stylesheet.css

```diff
@@ -365,7 +413,7 @@ html {
   border: 1px solid black;
 }
 
-img {
+img .related {
```

Comment:
Change to `img.related-img` (no spaces). This is what we agreed to with Kavi. You can very easily make this fix here in Github.

Reply:
change made